website/views/shared/modals/level-up.jade

```diff
@@ -16,6 +16,26 @@ script(type='text/ng-template', id='modals/levelUp.html')
       h4(style='margin-top: 1em')!=env.t('leveledUp', {level:'{{user.stats.lvl}}'})
       p=env.t('fullyHealed')
       br
+      div(ng-if='user.flags.classSelected && !user.preferences.disableClasses && !user.preferences.automaticAllocation')
+        a.btn.btn-default(data-toggle='collapse', data-target='#stat-allocation', aria-expanded=false, aria-controls='stat-allocation')=env.t('showHideQuickAllocation')
+        p &nbsp;
+        div.collapse#stat-allocation
+          table.table.text-left
+            tr
+              td
+                p(ng-if='::user.stats.lvl >= 100')!=env.t('noMoreAllocate')
+                p(ng-if='user.stats.points || user.stats.lvl < 100')
+                  strong.inline
+                    | {{user.stats.points}}&nbsp;
+                  strong.hint(popover-trigger='mouseenter', popover-placement='right', popover=env.t('quickAllocationLevelPopover'))=env.t('unallocated')
+              td
+            each statInfo, stat in { str: {title:"allocateStr",popover:'strengthText',allocatepop:'allocateStrPop'},int: {title:"allocateInt",popover:'intText',allocatepop:'allocateIntPop'},con: {title:"allocateCon",popover:'conText',allocatepop:'allocateConPop'},per: {title:"allocatePer",popover:'perText',allocatepop:'allocatePerPop'} }
```

Comment:
It would be great to have this `each` block in a mixin to avoid the code duplication here and in website/views/options/profile.jade. See website/views/shared/mixins.jade for some examples.